selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/SeleniumWebDriverHelper.java

```diff
@@ -65,4 +70,13 @@ public void waitAndClickOnElement(By elementLocator) {
   public void waitAndClickOnElement(WebElement webElement) {
     loadPageWait.until(visibilityOf(webElement)).click();
   }
+
+  public boolean elementIsVisible(By locator) {
+    try {
+      attachingElementToDomWait.until(visibilityOfElementLocated(locator));
+      return true;
+    } catch (TimeoutException ex) {
+      return false;
+    }
+  }
 }
```

Comment:
I have not quiet understood why we need SeleniumWebDriverHelper, why not SeleniumWebDriverUtils class? The classes very similar as for me.

Reply:
SeleniumWebDriverUtils - means utility to expand SeleniumWebDriver functionality itself, while SeleniumWebDriverHelper provides high-level functionality which just consume SeleniumWebDriver